src/desktop.md

```diff
@@ -275,86 +290,113 @@ $ flutter run -d linux
   the `-d <platform>` tag is optional.
 {{site.alert.end}}
 
-### Build a release app
+## Build a release app
 
-To generate a release build run one of the following commands:
+To generate a release build run one of the
+following commands:
 
 ```terminal
 $ flutter build windows
 $ flutter build macos
 $ flutter build linux
 ```
 
-### Distribution
-
-**In general, we don't recommend releasing a desktop app until
-desktop support is stable.**
-There are not yet full instructions, or tooling support,
-for making distributable applications. However,
-here is some information about how to use the current
-build output on other machines for testing purposes.
-
-#### Windows
-
-For building Windows executables, you can either use tooling to construct an
-MSIX installer, or you can build your own zip file that collects
-the components together.
-
-##### MSIX Packaging
-
-[MSIX][] is Microsoft's Windows app package format that provides a modern 
-packaging experience to all Windows apps. This format can either be used 
-to ship applications to Microsoft's Windows Apps store, or distribute 
-application installers directly.
-
-The easiest way to create an MSIX distribution for a Flutter project is to use
-the [`msix` pub package][msix package]. For an example of using the `msix` package
-from a Flutter desktop app, see the [Desktop Photo Search sample][].
+## Distribution
+
+**We don't recommend releasing a desktop
+app until desktop support is stable,**
+however, here is some information that you
+might still find useful.
+
+### Windows
+
+There are various approaches you can use for
+distributing your Windows application.
```

Comment:
Here and elsewhere -- we should update all uses of _application_ to _app_ or the other way around once we've made the call above.